mne/beamformer/tests/test_resolution_matrix.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+# Authors: Olaf Hauk <olaf.hauk@mrc-cbu.cam.ac.uk>
+#
+# License: BSD (3-clause)
+"""
+Test computation of resolution matrix for LCMV beamformers.
+
+If noise and data covariance are the same, the LCMV beamformer weights should
+be the transpose of the leadfield matrix.
+"""
+
+from copy import deepcopy
+import os.path as op
+import numpy as np
+from numpy.testing import assert_allclose
+
+import mne
+from mne.datasets import testing
+from mne.beamformer import make_lcmv, make_lcmv_resolution_matrix
+
+data_path = testing.data_path(download=False)
+subjects_dir = op.join(data_path, 'subjects')
+fname_inv = op.join(data_path, 'MEG', 'sample',
+                    'sample_audvis_trunc-meg-eeg-oct-6-meg-inv.fif')
+fname_evoked = op.join(data_path, 'MEG', 'sample',
+                       'sample_audvis_trunc-ave.fif')
+fname_raw = op.join(data_path, 'MEG', 'sample', 'sample_audvis_trunc_raw.fif')
+fname_fwd = op.join(data_path, 'MEG', 'sample',
+                    'sample_audvis_trunc-meg-eeg-oct-4-fwd.fif')
+fname_cov = op.join(data_path, 'MEG', 'sample', 'sample_audvis_trunc-cov.fif')
+
+
+@testing.requires_testing_data
+def test_resolution_matrix_lcmv():
+    """Test computation of resolution matrix for LCMV beamformers."""
+    # read forward solution
+    forward = mne.read_forward_solution(fname_fwd)
+
+    # remove bad channels
+    forward = mne.pick_channels_forward(forward, exclude='bads')
+
+    # forward operator with fixed source orientations
+    forward_fxd = mne.convert_forward_solution(forward, surf_ori=True,
+                                               force_fixed=True)
+
+    # evoked info
+    info = mne.io.read_info(fname_evoked)
+    mne.pick_info(info, mne.pick_types(info), copy=False)  # good MEG channels
+    info['projs'] = []
```

Comment:
can you add a comment why?

Reply:
> 
> 
> > @olafhauk did you see the comments from @agramfort above? It looks like they have not been addressed
> 
> I saw them after I replied to you. I've addressed them, but now I'm stuck with git problems again... Bear with me.

All tests look fine now. Please have a look at my comments and let me know if there is anything else to do for this PR.